.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*.{js,ts,tsx,jsx,json}]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
```

Comment:
still I can see this symbol "no new line" :P

Reply:
I'm afraid I don't know which setting you're referring to.  "no new line"? Can you point that our from https://editorconfig.org/ ?

The example given by you earlier was "insertFinalNewline" and that setting exists and was added as you can see above.

Reply:
thus symbol:
<img width="182" alt="Screenshot 2023-06-06 at 5 23 02 PM" src="https://github.com/SanjalKatiyar/fusion-console/assets/39404641/8f983cfd-bfdd-40a6-8bb2-ad9f8d93ec33">

Reply:
meaning this file needs a new line...

Reply:
merging the PR for now... we just need to add a new line manually here, we can do it in some other PR...

Reply:
> meaning this file needs a new line...

You should have started with this.

I added a new line to the file